spyder/plugins/layout/plugin.py

```diff
@@ -146,6 +148,43 @@ def on_toolbar_available(self):
             before=PreferencesActions.Show
         )
 
+    @on_plugin_teardown(plugin=Plugins.MainMenu)
+    def on_main_menu_teardown(self):
+        mainmenu = self.get_plugin(Plugins.MainMenu)
+        # Remove Panes related actions from the View application menu
+        panes_items = [
+            "plugins_menu",
+            LayoutContainerActions.LockDockwidgetsAndToolbars,
+            LayoutContainerActions.CloseCurrentDockwidget,
+            LayoutContainerActions.MaximizeCurrentDockwidget]
+        for panes_item in panes_items:
+            mainmenu.remove_item_from_application_menu(
+                panes_item,
+                menu_id=ApplicationMenus.View)
+        # Remove layouts menu from the View application menu
+        layout_items = [
+            'layouts_menu',

```

Comment:
This is also needs an enum. What about declaring something like

```python
class LayoutPluginMenus:
    PluginsMenu = "plugins_menu"
    LayoutsMenu = 'layouts_menu'
```

in the container of this plugin? And also using it there?